R/support.R

```diff
@@ -242,7 +270,13 @@ drop_trajectory_incidence <- function(obj) {
 
   assert_is(obj, "mcstate_trajectories")
   k <- grep("_inc$", rownames(obj$state))
-  obj$state <- obj$state[-k, , ]
+
+  if (length(dim(obj$state)) == 3) {
```

Comment:
if this sort of thing proliferates we should add an mcstate helper, something like

```
mcstate::array_subset(dim = 1, index = -k)
```